js/scripts/zone_loaders/village_zone_loaders.js

```diff
@@ -141,6 +141,12 @@ const loadVillageField = () => {
         false
     ));
 
+    GAME.addEntity(new FoodDrop(
```

Comment:
Nice. Do we want this in the village though?

Reply:
You'll be able to farm it too easily I'm thinking. What if the eel boss drops it or you hide it on the map in a place that's hard to get to.